extensions/spark/kyuubi-extension-spark-common/src/main/scala/org/apache/kyuubi/sql/KyuubiSQLConf.scala

```diff
@@ -284,4 +284,34 @@ object KyuubiSQLConf {
       .version("1.8.0")
       .intConf
       .createWithDefault(Int.MaxValue)
+
+  val GLUTEN_NON_SUPPORT_OPERATOR_LIST =
+    buildConf("spark.sql.gluten.nonSupportOperatorList")
```

Comment:
how about naming `spark.sql.gluten.fastFallbackOperators`? and as an open source projects, we'd better make the default value match a released version instead of the current main branch of Gluten

Reply:
+1, let's remove the default value and mark it as optional.